### PR TITLE
Upgraded Go/Echo to v2.

### PR DESCRIPTION
**Pull request by vishr**
Signed-off-by: Vishal Rana <vr@labstack.com>

### CODE REVIEW

**Comment by methane** [comment]
See https://github.com/TechEmpower/FrameworkBenchmarks/pull/1999/files#diff-354f30a63fb0907d4ad57269548329e3

There is no travis test for ehco.

File: frameworks/Go/echo/src/standard/main.go [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+	"github.com/labstack/echo"
+	"github.com/labstack/echo/engine/fasthttp"
+)
+
+func main() {
+	e := echo.New()
+	e.SetRenderer(common.Template)
+	common.InitRoutes(e)
+	common.InitPostgres()
+	e.Run(fasthttp.New(":8080"))
```
Comment (methane) [inline frameworks/Go/echo/src/standard/main.go]: fasthttp.New???

File: frameworks/Go/echo/src/fasthttp/main.go [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+		for _, ch := range children {
+			if err := ch.Wait(); err != nil {
+				log.Print(err)
+			}
+		}
+		os.Exit(0)
+		panic("unreachable")
+	}
+
+	runtime.GOMAXPROCS(1)
```
Comment (methane) [inline frameworks/Go/echo/src/fasthttp/main.go]: Why?

File: frameworks/Go/echo/src/standard/main.go (hunk 1 above)
Comment (vishr) [inline frameworks/Go/echo/src/standard/main.go]: Nice catch, will fix it.

File: frameworks/Go/echo/src/fasthttp/main.go (hunk 2 above)
Comment (vishr) [inline frameworks/Go/echo/src/fasthttp/main.go]: This code snippet is derived from `fasthttp-prefork` as it uses `reuseport` package.

Comment (methane) [inline frameworks/Go/echo/src/fasthttp/main.go]: I see.

**Comment by vishr** [comment]
- Travis included
- Fixed `standard.go` issue

**Comment by methane** [comment]
IMO, I don't like many Go frameworks using prefork.

Prefork is not usual nor ideal on Go.
go-prefork is just for reveals current Golang's runtime inefficiency on massive cores machine and very high I/O load.  It shouldn't be used to compare across frameworks.

But I understand feeling like "I want to win to stdlib's net/http !"
@vishr is echo author and @valyala is fasthttp author.  I respect them.

I hope future version of Go address this issue and all of go-prefork, fasthttp-prefork, and echo-prefork gone.
But this issue is only on abnormal usecase like /json and /plaintext.  Fixing it without any regression on
normal usecase is very hard.

**Comment by nbrady-techempower** [comment]
Hi @vishr we are getting ready to start our preview runs for round-13. In the meantime we are preparing to start work on verifying pull requests for round-14. Because of this, we are slightly restructuring our branches. Would you mind closing this and making a pull request that targets the new round-14 branch? Thanks!

Here's more information: http://frameworkbenchmarks.readthedocs.io/en/latest/Development/Contributing-Guide/#github-pull-request-procedure

**Comment by nbrady-techempower** [comment]
closed for #2103

**Pull request closed without merging**